evennia/settings_default.py

```diff
@@ -810,6 +810,28 @@
     {'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator'},
     {'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator'},
     {'NAME': 'evennia.server.validators.EvenniaPasswordValidator'}]
+    
+# Username validation plugins
+AUTH_USERNAME_VALIDATORS = [
+    {
```

Comment:
From a pure formatting perspective, we generally don't format dicts this way (javascript-like, maybe?), but tend to use a more compact form in this settings file.